docs/monitoring.md

```diff
@@ -0,0 +1,54 @@
+# Network Monitoring
+
+This document describes how to configure network monitoring within the node provided by `aemon` application.
+
+## Configuration
+
+By default monitoring is disabled. To turn it on `monitoring.publisher.pubkey` and `monitoring.active = true` need to be setup. To start posting transaction setup `monitoring.publisher.autostart` to `true`.
+
+To post transactions account's public and private key is needed. For online processing only public key needs to be configured. Offline post-processing can be done by filtering transaction by publisher public key.
+
+## Metrics configuration
+
+Monitoring uses statsd backend provided by [aec_metrics.erl](../apps/aecore/src/aec_metrics.erl). See metrics [configuration](../apps/aeutils/priv/epoch_config_schema.json#L545).
```

Comment:
This reference to line 545 is fragile. Could this be made absolute by rather naming the user config key?

Reply:
updated